client/reducers/auth.js

```diff
@@ -28,6 +29,18 @@ export default function (state = initState, action) {
         isAuthenticated: false,
         errorMessage: action.message
       }
+    case LOGOUT_REQUEST:
+      return {
+        ...state,
+        isFetching: true
+      }
+    case LOGOUT_SUCCESS:
+      return {
+        ...state,
+        isFetching: false,
+        isAuthenticated: action.isAuthenticated,
+        user: action.user
```

Comment:
add `user` to the initial state. don't use `action` for these values. just use `false` and `null`